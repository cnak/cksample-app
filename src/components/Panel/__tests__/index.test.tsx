import React from 'react';
import renderer from 'react-test-renderer';

import { Panel, PanelSubTitle, PanelTitle } from '../.';

describe('<Panel>', () => {
    it('matches snapshot', () => {
        const tree = renderer.create(<Panel />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    describe('<PanelTitle>', () => {
        it('matches snapshot', () => {
            const tree = renderer.create(<PanelTitle>This is a test</PanelTitle>).toJSON();
            expect(tree).toMatchSnapshot();
        });
    });

    describe('<PanelSubTitle>', () => {
        it('matches snapshot', () => {
            const tree = renderer.create(<PanelSubTitle>This is a test</PanelSubTitle>).toJSON();
            expect(tree).toMatchSnapshot();
        });
    });
});
