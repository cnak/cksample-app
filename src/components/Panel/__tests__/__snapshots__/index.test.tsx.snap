// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Panel> <PanelSubTitle> matches snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Helvetica",
        "fontSize": 18,
        "fontWeight": "bold",
        "lineHeight": 27,
        "marginBottom": 15,
        "textAlign": "center",
      },
    ]
  }
>
  This is a test
</Text>
`;

exports[`<Panel> <PanelTitle> matches snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Helvetica",
        "fontSize": 32,
        "fontWeight": "bold",
        "lineHeight": 35,
        "marginBottom": 20,
        "textAlign": "center",
      },
    ]
  }
>
  This is a test
</Text>
`;

exports[`<Panel> matches snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#28A197",
        "borderColor": "transparent",
        "borderStyle": "solid",
        "borderWidth": 5,
        "justifyContent": "center",
        "marginBottom": 10,
        "paddingBottom": 10,
        "paddingLeft": 10,
        "paddingRight": 10,
        "paddingTop": 10,
      },
    ]
  }
/>
`;
